go.mod

```diff
@@ -50,12 +49,12 @@ require (
 	github.com/opencontainers/runc v1.1.12
 	github.com/opencontainers/selinux v1.11.0
 	github.com/pkg/errors v0.9.1
-	github.com/pmezard/go-difflib v1.0.0
+	github.com/pmezard/go-difflib v1.0.1-0.20181226105442-5d4384ee4fb2
```

Comment:
is it intentional this is pulling in a version between tags? do we know if this library expects to be used on a non-tagged version and is stable there?

Reply:
I don’t think it’s intentional; the bump ultimately comes from https://github.com/hashicorp/consul where it was first done in https://github.com/hashicorp/consul/pull/15959 and then propagated.

There won’t be a non-tagged version of the library, it’s unmaintained; in fact [the difference between 1.0.0 and the commit referenced above](https://github.com/pmezard/go-difflib/compare/v1.0.0...master) is just a couple of documentation updates to fix an example and add a notice about the end of the project’s maintenance.